lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -193,6 +200,8 @@ def configure(conf)
       @common_labels = {}
       @common_labels.merge!(@labels) if @labels
 
+      @invalid_tag_regexp = %r{[^[:alpha:]\d/._-]}
```

Comment:
`invalid_tag_regexp` no longer exists though. I removed it and rely on url encoding instead.